packages/transactions/src/clarity/clarityValue.ts

```diff
@@ -169,3 +171,41 @@ export function getCVTypeString(val: ClarityValue): string {
       return `(string-utf8 ${utf8ToBytes(val.data).length})`;
   }
 }
+
+type ClarityTypetoValue = {
+  [ClarityType.OptionalNone]: NoneCV;
+  [ClarityType.OptionalSome]: SomeCV;
+  [ClarityType.ResponseOk]: ResponseOkCV;
+  [ClarityType.ResponseErr]: ResponseErrorCV;
+  [ClarityType.BoolTrue]: TrueCV;
+  [ClarityType.BoolFalse]: FalseCV;
+  [ClarityType.Int]: IntCV;
+  [ClarityType.UInt]: UIntCV;
+  [ClarityType.StringASCII]: StringAsciiCV;
+  [ClarityType.StringUTF8]: StringUtf8CV;
+  [ClarityType.PrincipalStandard]: StandardPrincipalCV;
+  [ClarityType.PrincipalContract]: ContractPrincipalCV;
+  [ClarityType.List]: ListCV;
+  [ClarityType.Tuple]: TupleCV;
+  [ClarityType.Buffer]: BufferCV;
+};
+
+/**
+ * @description narrow down the type of a generic ClarityValue
```

Comment:
```suggestion
 * Narrow down the type of a generic ClarityValue
```

There's no `@description` [tsdoc tag](https://tsdoc.org)